validation/js/src/main/scala/hmda/validation/rules/ts/syntactical/S028.scala

```diff
@@ -0,0 +1,15 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.dsl.{ Result, HmdaDsl, CommonDsl }
```

Comment:
We have a similar observation about `TsValidationEngine.scala`. The two versions have differences, but we suspect that they're not substantive. This duplication seems likely to cause maintenance headaches, so we'd like to avoid it if possible.